app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,24 @@
+class QuestionsController < ApplicationController
+  def index
+    @questions = Question.all
```

Comment:
新しい投稿が上に来るようにした方がよいでしょう。

```suggestion
    @questions = Question.order(id: :desc)
```